samples/nrf9160/lte_ble_gateway/README.rst

```diff
@@ -58,6 +58,25 @@ Two buttons and two switches are used to enter a pairing pattern to associate a
 
 When the connection is established, set switch 2 to **N.C.** to send simulated GNSS data to nRF Cloud once every 2 seconds.
 
+On the nRF9160 DK, the application state is indicated by the LEDs.
+
+LED 3 and LED 4:
+    * LED 3 blinking: The device is connecting to the LTE network.
+    * LED 3 ON: The device is connected to the LTE network.
+    * LED 4 blinking: The device is connecting to nRF Cloud.
+    * LED 3 and LED 4 blinking: The MQTT connection has been established and the user association procedure with nRF Cloud has been initiated.
+    * LED 4 ON: The device is connected and ready for sensor data transfer.
+
+    .. figure:: /images/nrf_cloud_led_states.svg
+       :alt: Application state indicated by LEDs
+
+    Application state indicated by LEDs
```

Comment:
```suggestion
```
You could delete these lines because they are stated in line 61, and I believe the lists that follow are just an extension of the one before.